alexandrie-index/src/tree.rs

```diff
@@ -74,17 +74,10 @@ impl Tree {
                 .lines()
                 .map(|line| Ok(json::from_str::<CrateVersion>(line?.as_str())?))
                 .collect::<Result<Vec<CrateVersion>, Error>>()?;
-            let latest = records
+            records
                 .into_iter()
                 .max_by(|k1, k2| k1.vers.cmp(&k2.vers))
                 .expect("at least one record should exist");
```

Comment:
You're not using this anymore, this will only calculate the max version. You can remove it all, and probably what's above too because it's unused now. I think the if can be left empty...

Reply:
yeah I saw that and go lazy, did not want to learn too much rust :D